x-pack/plugins/cloud/server/routes/chat.ts

```diff
@@ -33,7 +37,9 @@ export const registerChatRoute = ({
     },
     async (_context, request, response) => {
       const user = security.authc.getCurrentUser(request);
-      let { email: userEmail, username: userId } = user || {};
+      const { metadata, username } = user || {};
+      let userId = username;
+      let [userEmail] = (metadata as MetaWithSaml)?.saml_name || [];
```

Comment:
This is incorrect, based on what I'm seeing in Cloud... we need `saml_email`.  The token is coming back, but the AWS frame is rejecting it.  I'll update and redeploy to test.